### DISCUSSION TITLE
Label Sampler

### DISCUSSION THREAD

**Discussion by ghost** [TorchIO-project/torchio#910] [General]
Hello TorchIO Teams ! 

I have a question about Label Sampler utilization.. 
I work in segmentation of cerebral microbleeds ( so very very imabalanced data ), and i decided to use Label Sampler for extraction of subvolumes. 
But, the training loss decrease a little too fast, and i have the impression that is beacause the  label is always in center of the image. So, it's not very difficult for him to detect it. It's possible ? 

Thanks !

**Comment by fepegar** [comment]
Hi, @JonasZaoui. That's very hard to know without more information. How about your validation loss?

**Comment by romainVala** [comment]
The problem here is that using the Label Sampler with only one small ROI, then all patches will have the ROI in the center. (because of the construction of the Label Sampler).
When trying to learn from this scheme then the network need just to predict at volume center .
